data-es2016plus.js

```diff
@@ -37,6 +37,7 @@ exports.tests = [
           safaritp: true,
           webkit: true,
           duktape20: true,
+          duktape21: true,
```

Comment:
In any case the results are correct with the proposed change, so I'll apply that change to all the data files and update the pull. Thanks!